main/test/ca/uwaterloo/flix/language/debug/TestFormatType.scala

```diff
@@ -109,15 +128,37 @@ class TestFormatType extends FunSuite with TestUtils {
   test("FormatWellFormedType.Schema.External.02") {
     val latticeType1 = Type.mkLattice(List(Type.Str), loc)
     val latticeType2 = Type.mkLattice(List(Type.Int32, Type.Str), loc)
-    val restType = Type.KindedVar(5, Kind.Schema, loc, Rigidity.Flexible)
-    val tpe = Type.mkSchemaExtend(Name.Pred("A", loc), latticeType1, Type.mkSchemaExtend(Name.Pred("B", loc), latticeType2, restType, loc), loc)
+    val restType = Type.KindedVar(5, Kind.SchemaRow, loc, Rigidity.Flexible)
+    val tpe = Type.mkSchema(Type.mkSchemaRowExtend(Name.Pred("A", loc), latticeType1, Type.mkSchemaRowExtend(Name.Pred("B", loc), latticeType2, restType, loc), loc), loc)
 
     val expected = "#{ A<>(String), B<>(Int32, String) | '5 }"
     val actual = FormatType.formatType(tpe)(Audience.External)
 
     assert(actual == expected)
   }
 
+  test("FormatWellFormedType.SchemaRow.External.01") {
+    val relationType = Type.mkRelation(Type.Int32 :: Type.Str :: Nil, loc)
+    val tpe = Type.mkSchemaRowExtend(Name.Pred("S", loc), relationType, Type.SchemaRowEmpty, loc)
+
+    val expected = "#< S(Int32, String) >"
```

Comment:
Will this be `#(S(Int32, String)` or whats the plan actually?

Reply:
I'm going with that for now, unless there's a better idea.

Reply:
OK